keyboards/ergodox_ez/keymaps/colemak_es_osx/keymap.c

```diff
@@ -0,0 +1,411 @@
+/*
+  Copyright 2019 Mario Arias <mario@marioarias.org>
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 2 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+#include "keymap_spanish.h"
+
+enum layers {
+    BASE = 0,   //Colemak
+    QWERTY,     //Qwerty
+    FN,         //Colemak but FN1 to FN12 instead of numbers
+    NUM,        //Numpad
+    MOUSE,      //Mouse and media controls
+    IDEA        //Shortcuts for IDEA / Other tools
+};
+
+//Special paste
+#define S_PASTE     LSFT(LGUI(KC_V))
+//tmux prefix
+#define T_PREFIX    LCTL(KC_B)
+// Column mode
+#define I_COLUMN    ALGR(LCTL(LGUI(ES_MINS)))
+// Terminal
+#define I_TERM      ALGR(KC_F12)
+// Line comment
+#define I_LN_COM    LCTL(LGUI(KC_7))
+// Block comment
+#define I_BK_COM    LCTL(LGUI(KC_8))
+// Reformat code
+#define I_REFORM    LALT(LGUI(KC_L))
+// Rename
+#define I_RENAME    LSFT(KC_F6)
+// Find usages
+#define I_FUSAGE    LALT(KC_F7)
+// Code for Packt
+#define P_CODE      LCTL(ALGR(KC_X))
+// Search
+#define I_SEARCH    LCTL(LSFT(KC_F))
+
+
+enum custom_keycodes {
+  EPRM = SAFE_RANGE,
```

Comment:
This keycode isn't needed anymore. It can and should be replaced with `EEP_RST`.

Reply:
Fixed